### ISSUE TITLE
[Visual Requirements - Adaptive Cards - Homepage]: Tooltip is not defined for visual icons(New Card, Save File) Buttons over header section. 

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#7004]
https://microsoft.visualstudio.com/OS/_workitems/edit/31567724

**Comment by jwoo-msft** [comment]
should be very similar to what we have done for the XAML renderer tooltip feature.
